reana_cluster/config.py

```diff
@@ -40,3 +40,6 @@
     'kubernetes': KubernetesBackend,
 }
 """Dictionary to extend REANA cluster with new cluster backend."""
+
+reana_env_exportable_components = ['reana-server']
```

Comment:
Hmm, it is not "components" that are "exportable", so to speak, only information about them.  Having such a global variable may be a bit misleading to future users... Anyway, not a big deal if more descriptive names sound too long or not that great either...

Reply:
I didn't want to leave the magic value inside the `reana-cluster eval` command code, but I can move it back there. Another solution would be to add it as an option so by default it is `reana-server` and if you want more you could add `reana-cluster env --components reana-server wdb` but I think that this would make things more difficult rather than helping ...